src/reducers/schedule.js

```diff
@@ -31,10 +31,22 @@ function handleAddSection(state, { section }) {
   });
 }
 
+function handleRemoveSection(state, { sectionId }) {
```

Comment:
`return state.update('sections', sections => sections.filter(section => section.id !== sectionId));`
BOOM.

In all seriousness, `update` is a useful immutable method to know. Helps us write one liners like these.

Reply:
Oh shoot, you right